components/CompactFormationCard.tsx

```diff
@@ -0,0 +1,80 @@
+"use client";
+
+import Link from "next/link";
+import Image from "next/image";
+
+import { cn } from "@/lib/utils";
+import { type CmsData } from "@/lib/cms-types";
+import { type FormationData } from "@/lib/formations";
+
+import { Avatar, AvatarImage, AvatarFallback } from "@/components/ui/avatar";
+import { Card, CardContent } from "@/components/ui/card";
+
+type FormationCardProps = {
+  data: FormationData;
+  cmsData: CmsData;
+  className?: string;
+};
+
+export default function CompactFormationCard({
+  data,
+  className,
+  cmsData,
+}: FormationCardProps) {
+  const { id, formation, artifact, layout, username, user_image, name } = data;
+
+  const formationUrl = `/formations/${data.id}`;
+  const userUrl = `/users/${username}`;
```

Comment:
can we URL encode this value? should fix the "Mr. City" bug that is on production right now as a happy side effect, at least for the compact card :)